api/backend/ecs/service_manager.go

```diff
@@ -40,21 +41,25 @@ func NewECSServiceManager(
 	}
 }
 
-func (this *ECSServiceManager) ListServices() ([]*models.Service, error) {
-	serviceARNs, err := this.ECS.Helper_ListServices(id.PREFIX)
+func (this *ECSServiceManager) ListServices() ([]id.ECSServiceID, error) {
```

Comment:
would it be worth renaming this to `ListServiceIDS()`?

Reply:
I think that would just be more tedious than anything tbh

Reply:
It is more tedious, but as it stands, it's a bit inconsistent with similar methods that do list operations --- we usually return a model, and here's we're not.